async-nats/src/jetstream/account.rs

```diff
@@ -0,0 +1,59 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Default, Serialize, Deserialize, Clone, Copy, PartialEq, Eq)]
+pub struct Limits {
+    /// Maximum memory for this account (-1 if no limit)
+    pub max_memory: i64,
```

Comment:
Wouldn't it be more Rusty if we would map `-1` to `None`? 

Same for all those values.

Reply:
> Wouldn't it be more Rusty if we would map -1 to None?

It would.

Reply:
Is that an agreement to change it to that 😄 ?

Reply:
Yeah I'll fix it up today along with documenting the rest of the fields, these were carried over from the old definition.